app/javascript/packs/cms.ts

```diff
@@ -315,7 +315,7 @@ function setUpDropdownButtonOpen () {
   $(document).on('click', '.dropdown-toggle', (event) => {
     closeAllDropdowns(event.target as HTMLButtonElement)
 
-    $(event.currentTarget).siblings('.dropdown').addClass('expanded')
+    $(event.currentTarget).siblings('.dropdown').toggleClass('expanded')
```

Comment:
This fixes the item 2. mentioned here: https://github.com/3scale/porta/pull/4009#issuecomment-2651084904